solana/program_v2/programs/gateway_v2/src/instructions/admin/close_network.rs

```diff
@@ -17,7 +18,10 @@ pub struct CloseNetworkAccount<'info> {
     #[account(
         mut,
         close = destination,
-        constraint = network.can_access(&authority, NetworkKeyFlags::AUTH),
+        seeds = [NETWORK_SEED, network.authority.key().as_ref(), &network.network_index.to_le_bytes()],
+        bump,
+        // TODO: @william commented constraint for now
```

Comment:
these constraint issues have been fixed (check below as there are a bunch of these)